src/oc/hoc.c

```diff
@@ -36,6 +36,10 @@ void (*p_nrnpython_finalize)();
 int nrn_inpython_;
 int (*p_nrnpy_pyrun)(const char* fname);
 
+#if 0 /* defined by cmake if rl_event_hook is not available */
+#define use_rl_getc_function
+#endif
```

Comment:
I didn't understand the purpose of this if it's `#if 0`. Should this be just removed?